### ISSUE TITLE
role="tree" on navigation accordion menus is bad for accessibility, should be removed/changed 

### ISSUE THREAD

**Issue by admSI** [foundation/foundation-sites#12185]
## What is wrong now ?
Currently the ul element for an accordion menu is given the role "tree". This has a significant negative impact on accessibility via screen readers because many screen reader users navigate via links only, e.g. the "Find the next link" command in voiceover for Mac, and any links inside `role="tree"` are completely passed over when navigating this way. 

## Possible solution
Don't add `role="tree"` to accordion menus

## Context
After accessibility testing our site (which uses the accordion menu for it's main navigation) on multiple visually impaired users using different screen readers, a number of them were completely unaware of the existence of the navigation links until it was suggested they use tabbing to navigate rather than going from link to link as they often would.

- Foundation version(s) used: 6.6.3
- Browser(s) name and version(s): Chrome
- Device, Operating System and version: Mac/Windows

## Checklist

- [x ] I have read and follow the CONTRIBUTING.md document.
- [x ] There are no other issues similar to this one.
- [x ] The issue title and template are correctly filled.

**Comment by totallyspun** [comment]
I am also exploring this same issue. Raised by our client on Windows/JAWS and I can confirm also on Mac/VoiceOver.

VoiceOver announces the wrapping Navigation, and then next item I step into announces as a table. If I step through the actual items, it does not read the titles of the links.

I have applied the following workaround: Remove the role attribute on all <ul> and <li> elements in this component after Foundation has initializated it. VoiceOver steps through the nested list and reads titles and toggles as expected.
